version-metadata/action.yml

```
@@ -10,6 +10,19 @@ inputs:
   token:
     description: GitHub token used for authentication (you can use secrets.GITHUB_TOKEN)
     required: false
+  versionExtractionOverride:
```

```suggestion
  version-extraction-override:
```
Is it acceptable to use `kebab-case` [like](https://github.com/actions/checkout/blob/main/action.yml) [everybody](https://github.com/actions/setup-node/blob/main/action.yml) [else](https://github.com/docker/metadata-action/blob/master/action.yml)?